request.js

```diff
@@ -1,16 +1,15 @@
-var request = require('request');
-var qs = require('querystring');
-var messages = require('elasticio-node').messages;
-var debug = require('debug')('webhook:request');
+const request = require('request');
+const qs = require('querystring');
+const messages = require('elasticio-node').messages;
 
 exports.putOrPost = function putOrPost(method, msg, conf) {
     'use strict';
-    var uri = conf.uri;
-    var body = msg.body;
+    const uri = conf.uri;
+    const body = msg.body;
 
-    debug('Request body: %j', body);
+    this.logger.info('Request body: %j', body);
```

Comment:
Should be devided to 2 logs: info log about making request and trace log with request body